IGI.Lab2/src/CarPark.DAL/Interfaces/IEntity.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace CarPark.DAL.Interfaces
+{
+    public interface IEntity
+    {
+        public int Id { get; set; }
```

Comment:
Remove access modificator in interface methods.

Reply:
fixed